clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -1633,7 +1633,8 @@ default ListTransactionsResult listTransactions() {
      * coordinators in the cluster and collect the state of all transactions. Users
      * should typically attempt to reduce the size of the result set using
      * {@link ListTransactionsOptions#filterProducerIds(Collection)} or
-     * {@link ListTransactionsOptions#filterStates(Collection)}
+     * {@link ListTransactionsOptions#filterStates(Collection)} or
```

Comment:
Maybe a silly question, but I noticed this comment mentions filtering by producer ID or state. Do you know where that is done? It looks like durationFilter is the only subparser argument for list in TransactionCommand

Reply:
The ListTransactions api itself is able to filter by producer ids and states (see `handleListTransactions` in TransactionCoordinator). However, the cli tool does not support those filters currently.